pip/main.ts

```diff
@@ -52,13 +52,36 @@ const installStatus = PhylumApi.runSandboxed({
   args: Deno.args,
   exceptions: {
     run: ["./", "/bin", "/usr/bin", "/usr/local/bin", "~/.pyenv"],
-    write: ["./", "~/Library/Caches", "~/.cache", "~/.local", "~/.pyenv", "/tmp"],
-    read: ["~/Library/Caches", "~/.cache", "~/.local", "~/.pyenv", "/tmp", "/etc/passwd", "/etc/apache2/mime.types"],
+    write: ["./", "~/Library/Caches", "~/Library/Python", "~/.cache", "~/.local", "~/.pyenv", "/tmp"],
+    read: ["~/Library/Caches", "~/Library/Python", "~/.cache", "~/.local", "~/.pyenv", "/tmp", "/etc/passwd", "/etc/apache2/mime.types"],
     net: true,
   },
 });
 Deno.exit(installStatus.code);
 
+// Logs any identified issues to the screen.
+async function logIssues(jobStatus) {
+  for(const key in jobStatus.packages) {
+    const pkg = jobStatus.packages[key];
+    const issues = pkg.issues;
+
+    for(const idx in issues) {
+      const issue = issues[idx];
+      let severity = issue.severity.toLowerCase();
+      
+      if(["high", "critical"].indexOf(severity) != -1) {
```

Comment:
JavaScript's `contains` method is `includes` iirc. I always get this wrong myself and it's a horrible name, but it should work here.